src/System.IO.FileSystem.Watcher/src/System/IO/PatternMatcher.cs

```diff
@@ -140,7 +140,9 @@ public static bool StrictMatchPattern(string expression, string name)
                 // if name is shorter that the stuff to the right of * in expression, we don't
                 // need to do the string compare, otherwise we compare rightlength characters
                 // and the end of both strings.
-                if (name.Length >= rightLength && String.Compare(expression, 1, name, name.Length - rightLength, rightLength, StringComparison.OrdinalIgnoreCase) == 0)
+                if (name.Length >= rightLength && 
+                    String.Compare(expression, 1, name, name.Length - rightLength, rightLength, 
```

Comment:
`string`
However I'd prefer to avoid  c67e17d and leave it for [dotnet/codeformatter](https://github.com/dotnet/codeformatter)